rfc/0000-rfc-process.md

```diff
@@ -0,0 +1,59 @@
+---
+name: rfc_process
+started: 2024-07-04
+pr: pact-foundation/roadmap#0000
+---
+## Summary
+
+The Request for Comments (RFC) process is a way to propose and discuss changes to the Pact specification, ecosystem, and community. The RFC process ensures that changes are well-considered, documented, and communicated to all stakeholders. This RFC proposes the process itself, including how to submit an RFC, how to review and discuss it, and how to accept or reject it.
+
+## Motivation
+
+While the evolution of Pact and its ecosystem is open, the process for proposing and discussing changes is not well-defined. By introducing an RFC process, we can ensure that changes are well-considered, documented, and communicated to all stakeholders. This will help to ensure that changes are made in a transparent and collaborative way, and that the community has a voice in the direction of the Pact ecosystem.
+
+## Guide-level explanation
+
+### Scope of RFCs
+
+The Pact ecosystem is broad and includes, broadly speaking, the following areas:
+
+- The Pact specification itself,
+- The Pact broker,
+- The Pact CLI,
+- The Pact FFI,
+- The Pact implementation in various languages,
+
+This RFC process is intended to cover any change which will impact the broader Pact ecosystem and therefore includes the first four areas listed above.
+
+While specific changes to the Pact implementation in various languages are out of scope, we do wish to consider how changes in the first four areas will impact the implementation in various languages. Furthermore, we may wish to introduce or formalize common conventions or guidelines that are applied across the various languages to ensure consistency, interoperability, and familiarity for users.
```

Comment:
> various languages are out of scope 

may just be worth clarifying via this RFC process. See the individual contributing guidelines on the respective language repo?